test/controllers/conclusion_reports_controller_test.rb

```diff
@@ -131,6 +131,29 @@ class ConclusionReportsControllerTest < ActionController::TestCase
     assert_response :success
     assert_not_nil assigns(:filters)
     assert_equal 2, assigns(:filters).count
+    assert_equal 2, assigns(:conclusion_reviews).count
+    assert_template 'conclusion_reports/review_stats_report'
+  end
+
+  test 'filtered review stats report for scored reports' do
+    reviews(:past_review).update_column(:score_type, :none)
```

Comment:
Te copaste con los símbolos, sería más _adecuado_:

```suggestion
    reviews(:past_review).update_column(:score_type, 'none')
```

Reply:
y con un seattle ;-)

Reply:
De una, se me pasó el detalle =P